NativeDemo/TokamakDemo.xcodeproj/project.pbxproj

```diff
@@ -450,6 +453,7 @@
 			buildActionMask = 2147483647;
 			files = (
 				D1E5FDAD24C1D57000E7485E /* TokamakShim.swift in Sources */,
+				63F2511826BC0CD400CAE212 /* TokamakDOMAdditions.swift in Sources */,
```

Comment:
Do we need this in the native demo project? I imagine it's completely unused there, right?

Reply:
The way I've implemented this is by adding new constructors to `Text`, located in `TokamakStaticHTML`. The Demo uses this new functionality. However, since non-HTML environments don't see these new constructors, the Demo will no longer compile.

The solution is to add dummy functions into TokamakShim. That's the purpose of TokamakDOMAdditions.swift. These don't actually sanitize (as it is unnecessary in non-HTML environments); they just return a normal `Text`.

Reply:
Makes sense, thanks!